src/PointwiseFunctions/Hydro/InitialData/IrrotationalBns.cpp

```diff
@@ -172,7 +142,8 @@ void derivative_spatial_rotational_killing_vector(
     const tnsr::I<DataType, 3>& x,
     const Scalar<DataType>& /*local_angular_velocity_around_z*/,
     const Scalar<DataType>& /*sqrt_det_spatial_metric*/) {
-  *result = make_with_value<tnsr::iJ<DataType, 3>>(get<0>(x), 0.0);
+  set_number_of_grid_points(result, x);
+  tenex::update<ti::i, ti::J>(result, 0.0 * (*result)(ti::i, ti::J));
```

Comment:
The values in `result` may be uninintialized or NaN, so you can't multiply by them.  I don't know if we have a standard way to zero a Tensor, but `std::fill(result->begin(), result->end(), 0.0)` should work.